### PR TITLE
Add modals windows.

### PR DESCRIPTION
**Pull request by mercymugambi**
## On this milestone

1. When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
2. When the user clicks (or taps) the close (X) button, the popup disappears.
3. Use a JavaScript array to store all of the information for all projects.
4. For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
5. Update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. 
6. Mobile and desktop popup.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -11,3 +12,441 @@ document.querySelectorAll('.a-links').forEach((n) => n
...
+    counter1: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    role: 'Back End Dev',
+    skills: ['HTML', 'Ruby on rails', 'CSS', 'Javascript'],
+    counter2: '<img src="./assets/images/Counter.png" alt="Counter.png" />',
+    year: '2015',
+    sourceLink: 'https://github.com/mercymugambi/portfolio',
+    demoLink: 'https://mercymugambi.github.io/portfolio/',
+  },
+];
+
```
Comment (CollinsTatang) [inline js/main.js]: Hi @mercymugambi 
Great joy so far implementing this project requirement up to this extent.

 The project has a problem haer
 

- [ ] Please we need to have a single array of objects to store JavaScript objects containing the information for each project.

**Let reason together**
Imagine this was a database table storing your project data, will you create multiple tables to store data at different screen-size? I strongly believe your answer must be `NO`
We need just one database table to store our project data with their different `id:` and write CSS to make our screen display [responsive](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)

- [ ] Please remember to refactor your logical code below

File: style.css [hunk 2]
```diff
@@ -497,6 +499,117 @@ textarea::placeholder {
...
+  background: #fff;
+  border: 1px solid #7f8cff;
+  display: flex;
+  padding: 10px 23px;
+  align-items: center;
+}
+
+/* .desktop-popup-window {
+  display: none;
+} */
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

File: style.css [hunk 3]
```diff
@@ -756,4 +889,141 @@ textarea::placeholder {
   .arrows {
     width: 2px;
   }
+
+  /* .mobile-popup-window {
+  display: none;
+} */
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mercymugambi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mercymugambi** [comment]
Changes as requested above, I have worked on them

**Review by divinecharlotte — APPROVED** [review]
# Hello team @mercymugambi  and @MarwanDev   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- Good communication with the reviewer ✔️
- Descriptive PR ✔️
- Descriptive commit messages 👍 





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**